src/main/java/domain/RandomLottoGenerator.java

```diff
@@ -0,0 +1,29 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class RandomLottoGenerator implements LottoGenerator {
+	private static final List<Integer> numbers =
+		IntStream.rangeClosed(Lotto.LOTTO_LOWER_BOUND, Lotto.LOTTO_UPPER_BOUND)
+			.boxed().collect(Collectors.toList());
+
+	@Override
+	public Lottos generateLottos(int lottoCount) {
+		List<Lotto> lottos = new ArrayList<>();
+		for (int i = 0; i < lottoCount; i++) {
+			lottos.add(generateLotto());
+		}
+		return new Lottos(lottos);
```

Comment:
아래와 같은 방식으로 변경해볼 수 있겠네요. 😉
```suggestion
		return Stream.generate(this::generateLotto)
			.limit(lottoCount)
			.collect(Collectors.collectingAndThen(Collectors.toList(), Lottos::new));
```